pkg/koordlet/statesinformer/impl/config.go

```diff
@@ -34,6 +34,7 @@ type Config struct {
 	EnableNodeMetricReport      bool
 	MetricReportInterval        time.Duration // Deprecated
 	EnablePodTaskIds            bool
+	HookedPodResourceSocketPath string
```

Comment:
please move the parameter to the package pkg/koordlet/util/system or pkg/koordlet/util/kubelet